src/main/java/tests/ShopServiceTest.java

```diff
@@ -0,0 +1,110 @@
+package tests;
+
+import core.basesyntax.*;
```

Comment:
Avoid using wildcard imports. They can lead to unexpected behavior and make the code less readable as it's not clear which classes are used. Import only the classes that are needed.